doc/api/fs.md

```diff
@@ -4761,6 +4701,7 @@ The following constants are meant for use with the [`fs.Stats`][] object's
 [Caveats]: #fs_caveats
 [Common System Errors]: errors.html#errors_common_system_errors
 [FS Constants]: #fs_fs_constants_1
+[support of file system `flags`]: #fs_file_system_flags
```

Comment:
There are only 2 references here out of ASCII sorting, so let's do not increase the disorder and add this in the end if the list, after `[inode]:` and two other stray references.

Reply:
Resolved in 14da387a73a827670e5c379da727bbb489b93e3b